rotkehlchen/tests/db/test_evmtx.py

```diff
@@ -128,6 +131,65 @@ def test_add_get_evm_transactions(data_dir, username, sql_vm_instructions_cb):
         assert result == [tx1, tx3]
 
 
+@pytest.mark.parametrize('ethereum_manager_connect_at_start', [(INFURA_ETH_NODE,)])
+def test_get_or_create_transaction(ethereum_inquirer):
+    """Tests that get_or_create_transaction works fine. By testing get_or_create_transaction 
+    it checks that the requirements for a transaction are met before returning it."""
+    msg_aggregator = MessagesAggregator()
+    tx_hash = deserialize_evm_tx_hash(b'.h\xdd\x82\x85\x94\xeaq\xfe\n\xfc\xcf\xadwH\xc9\x0f\xfc\xd0\xf1\xad\xd4M\r$\x9b\xf7\x98\x87\xda\x93\x18')  # noqa: E501
+    with ethereum_inquirer.database.conn.write_ctx() as cursor:
+        ethereum_inquirer.database.add_blockchain_accounts(
+            write_cursor=cursor,
+            account_data=[
+                BlockchainAccountData(chain=SupportedBlockchain.ETHEREUM, address=ETH_ADDRESS1),
+                BlockchainAccountData(chain=SupportedBlockchain.ETHEREUM, address=ETH_ADDRESS2),
+                BlockchainAccountData(chain=SupportedBlockchain.ETHEREUM, address=ETH_ADDRESS3),
+            ],
+        )
+    tx = EvmTransaction(
+        tx_hash=tx_hash,
+        chain_id=ChainID.ETHEREUM,
+        timestamp=Timestamp(1451706400),
+        block_number=3,
+        from_address=ETH_ADDRESS2,
+        to_address=ETH_ADDRESS3,
+        value=FVal('4000000'),
+        gas=FVal('5000000'),
+        gas_price=FVal('2000000000'),
+        gas_used=FVal('25000000'),
+        input_data=MOCK_INPUT_DATA,
+        nonce=1,
+    )
+
+    with ethereum_inquirer.database.conn.write_ctx() as cursor:
+        dbevmtx = DBEvmTx(ethereum_inquirer.database)
+        # check that the transaction is properly added to the DB
+        returned_tx = dbevmtx.get_or_create_transaction(cursor, ethereum_inquirer, tx_hash, relevant_address=ETH_ADDRESS3)  # noqa: E501
+        errors = msg_aggregator.consume_errors()
+        warnings = msg_aggregator.consume_warnings()
+        assert len(errors) == 0
+        assert len(warnings) == 0
```

Comment:
shorten here and below

```suggestion
        assert(len(msg_aggregator.consume_errors()) == 0
        assert(len(msg_aggregator.consume_warnings()) == 0
```